src/server.ts

```diff
@@ -254,7 +336,7 @@ router.get('/owoembed', async (request: IRequest) => {
    We don't currently have custom profile cards yet,
    but it's something we might do. Maybe. */
 router.get('/:handle', profileRequest);
-router.get('/:handle/', profileRequest);
+router.get('/:handle/:language', profileRequest);
```

Comment:
We'd need to support translating profiles if we want to have this as a valid route tbh. I think Twitter does have an endpoint to translate a bio now?